### ISSUE TITLE
model checkpoint and training instruction 

### ISSUE THREAD

**Issue by YanghaoZYH** [silvercherry/Improving-Robust-Fairness-via-Balance-Adversarial-Training#1]
Dear authors,

Could you provide the trained checkpoint for the proposed method?

Or alternatively, can you provide the training script to reproduce the results in the paper? Seems that the default can not get the proper results shown in the paper.

Many thanks

**Comment by silvercherry** [comment]
Sorry, I haven't been following this related work for some time now. Could you align me with the code base of 'trades'? We have made modifications based on 'trades'

**Comment by ShuchunXu** [comment]
@YanghaoZYH Are you having a training loss value of 'nan'?

**Comment by YanghaoZYH** [comment]
@ShuchunXu  If I remember correctly, I did meet this problem, and I tuned the lr to 0.05/0.01 (I forgot which one), and then it resolved the nan but the performance is a bit far away from SOTA.

**Comment by ShuchunXu** [comment]
Thank you for your reply, I get it. Besides, i cannot understander the Equation 2. Can you help me understand it, thank you so much.
![屏幕截图 2024-09-20 085055](https://github.com/user-attachments/assets/4e5aa725-7ae4-4b1b-9898-c1387245f582)
